apps/web/pages/features/posts.tsx

```diff
@@ -0,0 +1,57 @@
+import GalleryRoute from '~/scenes/_Router/GalleryRoute';
+import { CmsTypes } from '~/scenes/ContentPages/cms_types';
+import PostsFeaturePage from '~/scenes/ContentPages/PostsFeaturePage';
+import { fetchSanityContent } from '~/utils/sanity';
+
+type Props = {
+  pageContent: CmsTypes.FeaturePage;
+};
+
+export default function PostsFeatureRoute({ pageContent }: Props) {
+  return <GalleryRoute element={<PostsFeaturePage pageContent={pageContent} />} navbar={false} />;
+}
+
+export const getServerSideProps = async () => {
+  const query = `
+  *[ _type == "featurePage" && id == "posts" ]{
```

Comment:
at first i was wondering what kind of graphql fuckery this was

Reply:
haha its sanity's query language GROQ which conceptually is very similar